emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/write/AbstractDynamoDBRecordWriter.java

```diff
@@ -118,7 +118,7 @@ public void write(K key, V value) throws IOException {
     if (result != null) {
       if (result.getConsumedCapacity() != null) {
         for (ConsumedCapacity consumedCapacity : result.getConsumedCapacity()) {
-          double consumedUnits = consumedCapacity.getCapacityUnits();
+          double consumedUnits = consumedCapacity.getTable().getCapacityUnits();
```

Comment:
In addition to the consumed capacity for table, I think we should also take LSI capacities into calculation since the provisioned throughput is shared by both table and LSIs while GSIs have their own throughputs. https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/LSI.html#LSI.ThroughputConsiderations

Reply:
@luyuanhao   thanks for pointing out to me, you are correct. Let me change the code and push it as per feedback..